src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -35,7 +37,8 @@ object SubmissionMonitorActor {
 
   /**
    * The response from querying the exec services.
-   * @param statusResponse If a successful response shows an unchanged status there
+    *
+    * @param statusResponse If a successful response shows an unchanged status there
```

Comment:
my intellij insists on alphabetising imports and making doc comments SUPER LEGIT and i don't know how to turn it off. for the record, i am not a horrendous pedant (in this particular case)